packages/flutter/lib/src/foundation/debug.dart

```diff
@@ -140,13 +140,15 @@ String? connectedVmServiceUri;
 /// [FlutterMemoryAllocations] should not increase size of the Flutter application
 /// if memory allocations are disabled.
 ///
+/// [flutterLibrary] is the name of the Flutter library where the object is declared.
+///
 /// Should be called only from within an assert.
 ///
 /// Returns true to make it easier to be wrapped into `assert`.
-bool debugMaybeDispatchObjectCreated(String library, String className, Object object) {
+bool debugMaybeDispatchCreated(String flutterLibrary, String className, Object object) {
```

Comment:
Why are we limiting this to flutter-internal use only? Wouldn't a third-party package potentially want to call use this as well?

Reply:
In leak tracker documentation to instrument custom objects I will suggest to either use helper from leak_tracker or, if they care about size, create their own helper.

Reply:
I do not see good solution that will be both compact for flutter and useable for third party packages.
I open for suggestions, if you have something in mind.